Docs: Improve readme (#115)
utopia-php/http: README.md

    <img height="45" src="docs/logo.png" alt="Logo">
</p>

[![Build Status](https://travis-ci.org/utopia-php/http.svg?branch=master)](https://travis-ci.org/utopia-php/http)
![Total Downloads](https://img.shields.io/packagist/dt/utopia-php/http.svg)
[![Discord](https://img.shields.io/discord/564160730845151244?label=discord)](https://discord.gg/GSeTUeA)

> Utopia Http is a PHP MVC based framework with minimal must-have features for professional, simple, advanced and secure web development. This library is maintained by the [Appwrite team](https://appwrite.io).

Reviewer: ```suggestion
Utopia HTTP is a PHP MVC based framework with minimal must-have features for professional, simple, advanced and secure web development. This library is maintained by the [Appwrite team](https://appwrite.io).
```

> Utopia Http is dependency-free. Any extra features, such as authentication or caching are available as standalone models in order to keep the framework core clean, light, and easy to learn.

Reviewer: ```suggestion
Utopia HTTP is dependency-free. Any extra features, such as authentication or caching are available as standalone models in order to keep the framework core clean, light, and easy to learn.
```

This is also not true anymore.

### Server Adapters

> The library supports server adapters to be able to run on any PHP setup. Youx1s could use the FPM server or the Swoole server.

Reviewer: ```suggestion
The library supports server adapters to be able to run on any PHP setup. You could use the FPM or Swoole server.
```

Resources are designed to prepare dependencies or context for the request. Resources are not meant to do functional logic or return callbacks. For such a use case, you should use hooks.

> To learn more about Http architecture and features, check out more in-depth [Getting started guide](/docs/Getting-Starting-Guide.md).

Reviewer: ```suggestion
To learn more about architecture and features for this library, check out more in-depth [Getting started guide](/docs/Getting-Starting-Guide.md).
```

## System Requirements

> Utopia Http requires PHP 8.0 or later. We recommend using the latest PHP version whenever possible.

Reviewer: ```suggestion
Utopia HTTP requires PHP 8.1 or later. We recommend using the latest PHP version whenever possible.
```

## More from Utopia

> Our ecosystem supports other thin PHP projects aiming to extend the core PHP Utopia Http.

Reviewer: ```suggestion
Our ecosystem supports other thin PHP projects aiming to extend the core PHP Utopia HTTP.
```
